client/app/autopilot.py

```diff
@@ -0,0 +1,50 @@
+from PIL import Image
+from io import BytesIO
+import numpy as np
+from picamera import PiCamera
+import tflite_runtime.interpreter as tflite
+
+
+class Autopilot():
+
+    def __init__(self, model_name, cam_res=None):
+
+        # camera
+        if cam_res is None:
+            cam_res = [224, 224]
+        self.camera = PiCamera()
+        self.camera.resolution = tuple(cam_res)
+
+        # model path
+        self.model_path = f"models/{model_name}"
```

Comment:
tiny change request: can you please pass the whole path to the class? so that the class doesn't have a path hardcoded :D

Reply:
done!